### DISCUSSION TITLE
Can I use @lexical/table in SvelteKit even if svelte-lexical don't support it ?

### DISCUSSION THREAD

**Discussion by neo-jgrec** [umaranis/svelte-lexical#89] [Q&A]
I'm doing a editor for a project and I would like to let the users to create a table easily, is there a way while still using svelte-lexical or do I need to use lexical in vanilla JS ? 

Also, I'm using SvelteKit

**Comment by umaranis** [comment]
Hi @neo-jgrec ,

The table functionality is not ported to svelte yet. 

As most of the functionality in lexical is implemented through react plugins including tables. It is not impossible to use it in vanilla JS (unless off course you include react as well).

**Reply by neo-jgrec** [reply]
Hey @umaranis ! 

Thank you for your answer.
I were wondering, if I'd like to contribute to svelte-lexical and add the implementation for tables, how could I do that ?
(Using another inside another framework sounds a little suspicious to me)

**Reply by umaranis** [reply]
Hi @neo-jgrec ,

It would be great to have you contribute. I can support you through the process. To get started, I would suggest reading: https://github.com/umaranis/svelte-lexical/blob/master/docs/porting-lexical-plugins.md

**Reply by neo-jgrec** [reply]
Thank you @umaranis,
I'll try to implement it, but it seem to be a heavy task !
I'll get you informed of my advancement !
 
 And if you have a Discord or something like this to communicate more easily, It would be nice

**Reply by umaranis** [reply]
Hi @neo-jgrec ,

Here is the discord link: https://discord.com/channels/953974421008293909/956476755856732160

**Reply by Sulkar** [reply]
Hello,
is there already a working group for the table plugin integration? The Discord link is not working. I am interested in contributing.

**Reply by umaranis** [reply]
Hi @Sulkar ,
There is currently no working group for the table plugin. 
It would be great if you could pick it up. I would be happy to support you during development.

Thanks :)
